app/kubernetes/views/resource-pools/create/createResourcePoolController.js

```diff
@@ -34,7 +34,7 @@ class KubernetesCreateResourcePoolController {
   onChangeIngressHostname() {
     const state = this.state.duplicates.ingressHosts;
     const hosts = _.flatMap(this.formValues.IngressClasses, 'Hosts');
-    const hostnames = _.map(hosts, 'Host');
+    const hostnames = _.compact(hosts.map((h) => h.Host));
```

Comment:
tip: non-lodash native es6 filter (for falsy values)
```suggestion
    const hostnames = hosts.map((h) => h.Host).filter((h) => !!h);
```
source:
- https://lodash.com/docs/4.17.15#compact
- https://www.measurethat.net/Benchmarks/Show/295/1/filter-vs-compact

Reply:
I actually prefer `_.compact` here as it's clearer what we do. I prefer native function wherever they're the same as the lodash helpers.

Reply:
Interesting. I had to look up compact as I didn't know what it was. 
Sounds good - matter of preference 👍